src/scss/_results.scss

```diff
@@ -1,3 +1,12 @@
+.warning-box {
+  border: 1px solid #01669b;
```

Comment:
I think we should start using variables for all the new css.

Reply:
Yes, done